industrial_ci/src/tests/source_tests.sh

```diff
@@ -139,11 +139,22 @@ function run_source_tests {
           catkin_lint_args+=(--strict -W2)
         fi
         ici_with_ws "$target_ws" ici_run "catkin_lint" ici_exec_in_workspace "$extend" "$target_ws"  catkin_lint --explain "${catkin_lint_args[@]}" src
-
     fi
     if [ "${CLANG_TIDY:-false}" != false ]; then
         run_clang_tidy_check "$target_ws"
     fi
+    if [ "${PYLINT:-false}" != false ]; then
+        local -a pylint_versions
+        ici_parse_env_array pylint_versions PYLINT_VERSIONS
+        local -a pylint_args
+        ici_parse_env_array pylint_args PYLINT_ARGS
+
+        ici_run "install_pylint" ici_install_pkgs_for_command pylint "${pylint_versions[@]}"
```

Comment:
This must be run for every element in `pylint_versions`, it should be moved into the for loop.